pyxlsb/workbook.py

```diff
@@ -41,14 +56,17 @@ def _parse(self):
       with self._zf.open('xl/sharedStrings.bin', 'r') as zf:
         temp.write(zf.read())
         temp.seek(0, os.SEEK_SET)
+      if self._debug:
+        print('Parsing xl/sharedStrings.bin')
       self.stringtable = StringTable(fp=temp)
     except KeyError:
       pass
 
   def get_sheet(self, idx, rels=False):
+    assert self._mode == 'r', 'Not opened for reading'
```

Comment:
Why does this have to be here? I want to read the sheet and edit a few cells. This is not helping